keep/api/models/db/migrations/versions/2025-03-18-14-54_971abbbf0a2c.py

```diff
@@ -0,0 +1,32 @@
+"""Default workflow_id=None for WorkflowExecution for test runs
+
+Revision ID: 971abbbf0a2c
+Revises: aff0128aa8f1
+Create Date: 2025-03-18 14:54:56.003392
+
+"""
+
+from alembic import op
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "971abbbf0a2c"
+down_revision = "aff0128aa8f1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    with op.batch_alter_table("workflowexecution", schema=None) as batch_op:
+        batch_op.alter_column(
+            "workflow_id", existing_type=mysql.VARCHAR(length=255), nullable=True
```

Comment:
workflow_id is a part of multiple keys/constraints in this table. Won't we have conflicts later with NULL value?

Reply:
No, was just curious =)